src/main/cxx/benchmark-runner/benchmark-runner.cxx

```diff
@@ -285,42 +282,42 @@ BENCHMARK_CAPTURE(BM_JNIPolyglotDistance, placeholder, "js",
     ->Setup(DoJNISetup)
     ->Teardown(DoJNITeardown);
 
-void centry_function(exec_mode mode, const char* language, const char* code,
+void centry_function(const exec_mode mode, const char* language, const char* code,
                      double a_lat, double a_long, double b_lat, double b_long) {
   graal_isolatethread_t* thread = create_isolate();
 
   switch (mode) {
     case CENTRY_JAVA_DISTANCE: {
-      printf("%.2f km\n", distance(thread, a_lat, a_long, b_lat, b_long));
+      std::printf("%.2f km\n", distance(thread, a_lat, a_long, b_lat, b_long));
       break;
     }
 
     case CENTRY_RUBY_DISTANCE: {
-      printf("%.2f km\n", distance_ruby(thread, a_lat, a_long, b_lat, b_long));
+      std::printf("%.2f km\n", distance_ruby(thread, a_lat, a_long, b_lat, b_long));
       break;
     }
 
     case CENTRY_POLYGLOT_DISTANCE: {
-      printf("%.2f km\n",
+      std::printf("%.2f km\n",
              distance_polyglot(thread, (char*)language, (char*)code, a_lat,
                                a_long, b_lat, b_long));
       break;
     }
 
     default: {
-      fprintf(stderr, "Unexpected mode encountered for CEntry calls: %d\n",
+      std::fprintf(std::stderr, "Unexpected mode encountered for CEntry calls: %d\n",
               mode);
-      exit(1);
+      std::exit(1);
```

Comment:
`std::exit` comes with some caveats which I _don't_ think apply in this file but here are the docs for future reference anyways: https://en.cppreference.com/w/cpp/utility/program/exit